cfme/infrastructure/host.py

```diff
@@ -663,8 +663,11 @@ def step(self):
         self.prerequisite_view.toolbar.monitoring.item_select('Utilization')
 
 
-def get_credentials_from_config(credential_config_name):
-    creds = conf.credentials[credential_config_name]
-    username = ("{}\\{}".format(creds.domain, creds.username) if creds.get('domain') else
-                creds.username)
-    return Host.Credential(principal=username, secret=creds["password"])
+def get_credentials_from_config(credential_configs):
+    creds = {}
+    for cred_type in credential_configs:
```

Comment:
**REQUIRED** use `for cred_type, cred in credential_configs.items()` instead of the dict access